src/docs/admin/references/NodeSourceStartupScriptReference.adoc

```diff
@@ -0,0 +1,41 @@
+Most cloud node source infrastructures allows the user to customize the VM startup script. 
```

Comment:
'Most node source of cloud infrastructure' sounds clearer to me.

Reply:
I agree `cloud node source infrastructures` is too long, but `node source of cloud infrastructure` seems also a bit strange to me... since the subject of "allow to customize startup script" is "cloud infrastructures" instead of "node source".
But since readers may not understand "cloud infrastructures" without mention "node source", so I prefer to keep it like this if I can't find a better term.